### PR TITLE
fix(ui): make select connector page ssr

### PR DESCRIPTION
**Pull request by kahboom**
## Description
This PR fixes the Next.js build, which was broken because I was previously using the `create-connector` page as a client component, when it was meant to be server-side. To do this, it meant a little bit of refactoring (see below). In addition, the fix for `@kaoto-next/ui` also seems to have gotten lost between the PR merges.

## Changes
- create a new `CatalogContainer` component to hold the `@kaoto-next/ui` `Catalog` component to make it easier to test and create stories for
- rename `CreateConnector` to `SelectConnector` to better align with UXD for MVP
- fix issue where `SelectConnector` page was meant to be a server-side rendered page but was actually a client page
- create a `SelectConnectorClient` page, which takes the business logic from `SelectConnectorPage` and assembles UI components like `CatalogContainer` and PatternFly components in one page
- create story for the new `SelectConnectorClient` page
- remove reference to kaoto styles (again)
- add support for Next.js default aliasing from within Storybook (i.e. now you can import components into stories like `import CatalogContainer from '@/components/Catalog/Catalog'` just as you can in the rest of the app)

In summary, this is the structure I'm trying to follow:
- A server-side rendered Next.js page called `SelectConnectorPage`, which contains the business logic and API calls
- A client-side rendered Next.js page called `SelectConnectorClient`, which receives the data from `SelectConnectorPage` and assembles dumb UI components
- Dumb UI components that should only do one thing, and should not really contain much in them
- A story for each of the client-side components and page

## Screenshots

The build works again:

Screen Shot 2023-10-06 at 4 57 28 pm

A story for the `SelectConnectorClient` page:

Screen Shot 2023-10-06 at 5 23 03 pm

A story for the `Catalog` UI component:

Screen Shot 2023-10-06 at 5 23 12 pm

### CODE REVIEW

**Review by kumaradityaraj — APPROVED** [review]

File: src/components/Catalog/Catalog.tsx
Comment (shivamG640) [inline src/components/Catalog/Catalog.tsx:1]: Would re-naming this file CatalogContainer make more sense?
Conventionally, the name of the functional component should match the file name.
